src/main/steps/c100-rebuild/miam/upload/template.njk

```diff
@@ -0,0 +1,98 @@
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/file-upload/macro.njk" import govukFileUpload %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+
+{% extends "common/page.njk" %}
+
+
+{% block page_content %}
+
+{% if FileErrors.length > 0 %}
+{{ govukErrorSummary({
+  titleText: "There is a problem",
+  errorList: FileErrors
+}) }}
+
+{%endif%}
+
+
+  <h1 class="govuk-heading-xl">{{title}} </h1>
+  <p class="govuk-body">{{ youNeed }}</p>
+  <p class="govuk-body">{{ youNeed2 }}</p>
+
+
+
+{% set uploadReqTest = ""%}
+
+{% for item in uploadRequirement %}
+    {% set uploadReqTest = uploadReqTest + "<li>" + item + "</li>"%}
+{% endfor %}
+
+{% set requirementTest = "<ol class='govuk-list govuk-list--number'>" + uploadReqTest + "</ol>" %}
+
+{{ govukDetails({
+  summaryText: uploadDescription,
+  text: requirementTest  | safe
+}) }}
+
+ <div class="govuk-form-group govuk-inset-text">
+   {% for error in sessionErrors %}
+    {%- if  error.propertyName === 'document'  %}<div class="govuk-form-group--error">  {% endif %}
+    {% endfor %}
+ <form action="{{postURL}}" method="post" enctype="multipart/form-data">
+ <input type="hidden" name="_csrf" id="csrfToken" value="{{ csrfToken }}">
+  {% for error in sessionErrors %}
+    {%- if  error.propertyName === 'document'  %}
+    {%- if  error.errorType === 'required'  %}
+         <span class="govuk-error-message" id="steps-children-parental-responsibility-form-parental-responsibility-error">
+    <span class="govuk-visually-hidden">Error: </span>{{errors['document'].required}}</span>
+     {% endif %}
+      {%- if  error.errorType === 'multipleFiles'  %}
+     <span class="govuk-error-message" id="steps-children-parental-responsibility-form-parental-responsibility-error">
```

Comment:
can we change the id as it might overlap once we integrate with child details?

Reply:
Done